libs/novui/src/global.styles.ts

```diff
@@ -14,6 +16,11 @@ export const GLOBAL_CSS = defineGlobalStyles({
 
     backgroundColor: 'surface.page',
     overflow: 'hidden',
+    scrollbarWidth: 'thin',
+    scrollbarColor: {
+      base: `${token('colors.legacy.B80')} transparent`,
+      _dark: `${token('colors.legacy.B30')} transparent`,
```

Comment:
Panda doesn't support token evaluation for CSS values requiring 2 or more properties to be specified inside a single string and doesn't have native support for these properties ([ref](https://panda-css.com/docs/utilities/interactivity)), so we need to use template literals instead.

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** This is also a good code comment!‏

Reply:
Also, we should make B80 / B30 into its own semantic color token for `scrollbar`, and then should be able to just do a single template literal here

Reply:
Great points, I've created new semantic tokens and updated to use those tokens throughout the global css file.